### ISSUE TITLE
nzbgeek api key format change

### ISSUE THREAD

**Issue by colaholiker** [SickChill/SickChill#6990]
What you did:
The api key format from nzbgeek has change. 
i have change to the new key

What happened:
The Api geek im Log and sabnzb has change and the download fails

In the config.ini:
[Newznab]
  newznab_data = "NZBGeek|https://api.nzbgeek.info/|lqC1DqtInRsTKtPIbDyuDsSXWCmATBqs|5030,5040|0|eponly|0|1|1"

In the log:
2021-01-26 21:09:32 INFO :: SEARCHQUEUE-BACKLOG-74936 :: [NZBGeek] :: Found result XXX at https://api.nzbgeek.info/api?t=get&id=XXX&apikey=57274af5fa67d19bdb51de7a3bca15d3

The Api is the orginial Key. Before submiting the issue i have change the key by nzbgeek.
Sorry for the bad english

**Comment by ronrosenfeld** [comment]
I had no issues.

I merely pasted the new API into the line in the `Configure custom newznab providers`; saved it, and all worked ok.

![image](https://user-images.githubusercontent.com/18516066/105991029-4c250e80-6071-11eb-8aa1-dd2a520a314b.png)

**Comment by colaholiker** [comment]
Which Option have you by 'Send .nzb files to'?
My Choice ist SABnzbd.

Other ideas?




  SickChill Info:
Install: pip
Version: v2020.11.24-1
Database Version: 44.1

  Python Version:
3.7.9 (default, Oct 27 2020, 14:36:14) [Clang 8.0.1 (tags/RELEASE_801/final 366581)]

  SSL Version:
OpenSSL 1.1.1h 22 Sep 2020

  OS:
FreeBSD-12.1-RELEASE-p10-amd64-64bit-ELF

  Locale:
en_US.UTF-8

**Comment by colaholiker** [comment]
Which Version are you useing?

Can the links come from the cache?
https://github.com/SickChill/SickChill/commit/6c85c7a434b340587f8ff01753b4e2ce0ddd990a

Where can I clean the SearchCache?

**Comment by colaholiker** [comment]
Yes. Cache

$ mv cache.db cache.db.20210127

Thanks 
I apologize for my bad english ;-)

**Comment by ronrosenfeld** [comment]
> Which Version are you useing?
> 
> Can the links come from the cache?
> [6c85c7a](https://github.com/SickChill/SickChill/commit/6c85c7a434b340587f8ff01753b4e2ce0ddd990a)
> 
> Where can I clean the SearchCache?

```
 SickChill Info:
Branch: master
Commit: c6ee7aa889661dde40ea2240ecb5d88da776e039
Version: v2020.11.24-1
Database Version: 44.5
  Python Version:3.7.9 (default, Oct 22 2020, 05:22:26) [GCC 9.3.0]
  SSL Version:OpenSSL 1.1.1g 21 Apr 2020
  OS:Linux-4.4.59+-x86_64-with
  Locale:en_US.UTF-8
```
and the API seems to be properly stored in `config.ini`

NZB's are sent to `nzbGet`

I am not aware of the interaction between cache.db and your problem.  As I wrote, I merely made the change on the appropriate configuration screen within Sickchill, and had no problems.

**Comment by kesawi** [comment]
I've experienced the same issue as @colaholiker. Updated the nzbGeek API key through the `Configure custom newznab providers` screen and saved it. The API key updated in `config.ini` however SickChill kept passing the old API key through to sabnzbd. Deleting the `cache.db` file and restarting SickChill resolved the issue.

I'm running the latest docker image from linuxserver.io `Digest:sha256:a980b174941e679e26da3a4172bfa1a0a9754dab13706f7b479906655c6883ba`

```SickChill Info:
Install: pip
Version: v2020.11.24-1
Database Version: 44.2

  Python Version: 3.8.5 (default, Jul 20 2020, 23:11:29) [GCC 9.3.0]

  SSL Version: OpenSSL 1.1.1i 8 Dec 2020

  OS: Linux-5.10.6-1.el7.elrepo.x86_64-x86_64-with
```

Also @colaholiker I suggest you change your nzbGeek API key if it is your real key as you've now publically posted it and anyone could use it to access your account (if you haven't enabled a password or 2FA on nzbGeek), or make API requests via your account.
